src/content/docs/reference/snippets/nodejs/RequestFields.ts

```diff
@@ -41,8 +42,15 @@ const server = http.createServer(async function (
     headers: req.headers,
   };
 
-  // @ts-ignore
-  const decision = await aj.protect({}, details);
+  // Provide a function to get the body in case it is needed by a rule.
+  const getBody = async () => {
+    return await readBody(req, {
+      // Only read up to 1 MiB of the body
+      limit: 1048576,
```

Comment:
Do you want the try/catch around this to return an undefined body in failure cases?